samples/bluetooth/mesh/chat/include/chat_cli.h

```diff
@@ -22,31 +22,28 @@ extern "C" {
 #endif
 
 /* .. include_startingpoint_chat_cli_rst_1 */
-/** Company ID of the Bluetooth Mesh Chat Client model. */
-#define BT_MESH_CHAT_CLI_VENDOR_COMPANY_ID    0x0059
-
 /** Model ID of the Bluetooth Mesh Chat Client model. */
 #define BT_MESH_CHAT_CLI_VENDOR_MODEL_ID      0x000A
 
 /** Non-private message opcode. */
 #define BT_MESH_CHAT_CLI_OP_MESSAGE BT_MESH_MODEL_OP_3(0x0A, \
-				       BT_MESH_CHAT_CLI_VENDOR_COMPANY_ID)
```

Comment:
I believe this should not be changed. We want to allow people to update/expand this model and use it for their own purpose and in that case, substitute this number with their own company ID. Applies to all changes in this file.